cmd/namespace/create.go

```diff
@@ -29,15 +29,17 @@ import (
 
 // CreateOptions represents the options that namespace create has
 type CreateOptions struct {
-	Members   *[]string
-	Namespace string
-	Show      bool
+	Members      *[]string
+	Namespace    string
+	Show         bool
+	SetCurrentNs bool
 }
 
 // Create creates a namespace
 func Create(ctx context.Context) *cobra.Command {
 	options := &CreateOptions{
-		Show: true,
+		Show:         false,
+		SetCurrentNs: false,
```

Comment:
I'd remove this line since it's already being set by the flag. Having it here just overwrites the flag